docs/en/advanced_guides/datasets/waymo.md

```diff
@@ -65,22 +64,16 @@ mmdetection3d
 │   │   │   ├── validation
 │   │   │   ├── testing
 │   │   │   ├── gt.bin
+│   │   │   ├── cam_gt.bin
```

Comment:
maybe we need to explain the differences among the three .bin file

Reply:
It will be add when we merge the PR of support mvfcos3d++.